src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -253,45 +250,42 @@ impl CopyInterpreter {
             Some(max_files)
         };
 
-        let mut all_source_file_infos = if force {
+        let all_source_file_infos = if force {
             StageTable::list_files(&stage_table_info, max_files).await?
         } else {
             StageTable::list_files(&stage_table_info, None).await?
         };
+        let num_all_files = all_source_file_infos.len();
 
-        info!("end to list files: {}", all_source_file_infos.len());
+        info!("end to list files: got {} files", num_all_files);
```

Comment:
It would be better to log the max_files.

Reply:
done